src/gui/torrentcontentmodel.cpp

```diff
@@ -29,9 +29,24 @@
  */
 
 #include <QDir>
+#include <QFileIconProvider>
+#ifdef Q_OS_WIN
+#include <Windows.h>
+#include <Shellapi.h>
+#ifndef NO_QWINEXTRAS
+#include <QtWin>
+#else
+Q_GUI_EXPORT QPixmap qt_pixmapFromWinHICON(HICON icon);
+#endif
+#else
+#include <QMimeDatabase>
+#include <QMimeType>
+#endif
+#include <QFileInfo>
```

Comment:
We didn't strict defined this in coding guidelines, but maybe it's better to place like this:
```
// headers that are needed in all conditions
#include <QFileIconProvider>

// header inclusion with conditions
#ifdef Q_OS_WIN
#include <Windows.h>
#endif

// project headers
#include "guiiconprovider.h"
```